### ISSUE TITLE
Quant_aware_training for mobilenet_v2

### ISSUE THREAD

**Issue by 441711335** [winfredsu/quant_aware_training#1]
Hello @winfredsu  did you have met the problem that while fine-tuning the trained mobilenet_v2 model the validation accuracy become to 0.1. It's far less than the validation accuracy in training procedures. How can I solve this problem?  
looking for your help !
thank you

**Comment by wangweihq8707** [comment]
你将 bn_折叠偏置添加16位量化，但是，你在转IR （https://github.com/winfredsu/TF2IR） 的程序里， 还是用的int8 （     layer['bias_dtype'] = 'int8'） 这样，到硬件实现上，是否会出现问题？
